aspnetcore/fundamentals/target-aspnetcore.md

```diff
@@ -0,0 +1,280 @@
+---
+title: Use ASP.NET Core APIs in a class library
+author: scottaddie
+description: Learn how to use ASP.NET Core APIs in a class library.
+ms.author: scaddie
+ms.custom: mvc
+ms.date: 12/16/2019
+no-loc: [Blazor]
+uid: fundamentals/target-aspnetcore
+---
+# Use ASP.NET Core APIs in a class library
+
+By [Scott Addie](https://github.com/scottaddie)
+
+This document provides guidance for using ASP.NET Core APIs in a class library. For all other library guidance, see [Open-source library guidance](/dotnet/standard/library-guidance/).
+
+## Determine which ASP.NET Core versions to support
+
+ASP.NET Core adheres to the [.NET Core support policy](https://dotnet.microsoft.com/platform/support/policy/dotnet-core). Consult the support policy when determining which ASP.NET Core versions to support in a library. A library should make an effort to:
+
+* Support all ASP.NET Core versions classified with *Long-Term Support* (LTS).
+* Ignore non-LTS ASP.NET Core versions, such as those classified with *End of Life* (EOL).
```

Comment:
This statement might be a little more restrictive than what I had in mind.

Possible rephrase: `Don't feel obligated to support ASP.NET Core versions classified as End of Life`

My thoughts:
 - We shouldn't tell people that it's **wrong** to support old versions. It's not really our problem if they want to spend their own time and effort on it. 
- I want library authors to be aware that EOL is a thing, find the relevant info, and then they can consider whether they want to support those versions.
- We definitely don't want to discourage library authors from targeting non-LTS releases, like 3.0. Library updates become adoption blockers for people to these releases really fast.

Reply:
Agree with this.